### DISCUSSION TITLE
Install to latest Raspberry Pi OS - success and a question about logs

### DISCUSSION THREAD

**Discussion by edllew** [mcguirepr89/BirdNET-Pi#592] [Q&A]
Hi,

I did an installation today to a Pi 4B starting from a fresh install of the latest Raspberry Pi OS, 64 bit, lite version.
I am up and running now, and it seems to work.  I am getting bird detections.  Very cool.
I saw a note saying the latest OS update hadn't been tested, but so far so good.

But my first installation attempt failed part way through, asking me to do an additional install:
sudo apt-get install python3-venv
So I reinstalled my OS, and did that additional install (python3-venv) before the main birdnet install.
Then all was good.

But I do notice that my system logs are filling up quickly with BirdNet messages.  Is there a way to turn off or turn down logging from BirdNet?  I want to save my SD card.  I installed log2ram, redirected some of the intermediate files to /dev/shm/, and may later start running from an SSD or HDD.

Looking around I see in the web gui..Tools..Services..BirdNET Log.    I turned that service off, rebooted, verify it is still off, and now the web gui no longer shows the log under "View Log", but /var/log/syslog and /var/log/daemon.log are still getting many messages from BirdNet.

-Ed

**Comment by mcguirepr89** [comment]
Hi, @edllew -- logging is handled by `systemd-journald.service` and its socket. I _think_ you can just disable (and mask) the logging entirely with `sudo systemctl disable --now systemd-journald.service systemd-journald.socket && sudo systemctl mask systemd-journald.service systemd-journald.socket`

Give that a test and let me know if it achieves what you're looking for.
